src/main/java/org/opensearch/security/privileges/ProtectedIndexAccessEvaluator.java

```diff
@@ -83,8 +84,8 @@ public PrivilegesEvaluatorResponse evaluate(final ActionRequest request, final T
             presponse.allowed = false;
             return presponse.markComplete();
         }
-        if((indexMatcher.matchAny(requestedResolved.getAllIndices())
-                || requestedResolved.isLocalAll())
+        if((requestedResolved.isLocalAll()
```

Comment:
any reason to change the sequence?

Reply:
isLocalAll() need to be called first before getAllIndices()